internal/handler/Handler.go

```diff
@@ -0,0 +1,281 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/VTerenya/employees/internal"
+	"github.com/VTerenya/employees/internal/service"
+	"net/http"
+	"regexp"
+)
+
+var (
+	positionsRe      = regexp.MustCompile(`^\/positions[\/]*$`)
+	positionRe       = regexp.MustCompile(`^\/position\/(\d+)$`)
+	createPositionRe = regexp.MustCompile(`^\/position$`)
+
+	employeesRe      = regexp.MustCompile(`^\/employees[\/]*$`)
+	employeeRe       = regexp.MustCompile(`^\/employee\/(\d+)$`)
+	createEmployeeRe = regexp.MustCompile(`^\/employee$`)
+)
+
+type Handler struct {
+	service service.ServiceHandler
+}
+
+func NewHandler(ser *service.Service) *Handler {
```

Comment:
Here the interface should used, You'll can't test that package properly if you use exact struct as argument.